test/verify/check-machines

```diff
@@ -2484,6 +2495,27 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
                 ._deleteVm(dialog) \
                 .checkEnvIsEmpty()
 
+        def createThenDeleteInInstallation(self, dialog, failed, error_msg):
+            name = dialog.name
+
+            dialog.open().fill()
+            self._create(dialog)
+
+            if not dialog.start_vm:
+                self.browser.click("#vm-{}-install".format(name))
+                self._assertVmStates(name, "creating VM installation", "running")
+
+            self.browser.click("#vm-{}-delete".format(name))
+            self.browser.click("#vm-{}-delete-modal-dialog button.pf-c-button.pf-m-danger".format(name))
+            self.browser.wait_not_present("#vm-{}-delete-modal-dialog".format(name))
+
+            if failed:
+                self.browser.wait_present("#app div[aria-label=\"Danger Alert\"]")
```

Comment:
It would be nice to be a function: 
``` diff
+        def checkErrorNotification(self, error_msg):
+            b = self.browser
+            b.click(".toast-notifications-list-pf div.pf-c-alert .more-button")
+            self.browser.wait_in_text("#app div.pf-c-alert__description", error_msg)
```

we can to reuse it in other places as well.